src/main/java/uk/gov/hmcts/reform/professionalapi/repository/OrganisationRepository.java

```diff
@@ -44,6 +52,14 @@ public interface OrganisationRepository extends JpaRepository<Organisation, UUID
     @EntityGraph(value = "Organisation.alljoins")
     List<Organisation> findAll();
 
+
+    @Query("SELECT o FROM Organisation o")
+    List<Organisation> findAllWithoutJoins();
```

Comment:
is this query is not using joins?

Reply:
No, remember we discussed this yesterday and looked at the sql produced

Reply:
Without the `@EntityGraph(value = "Organisation.alljoins")` I was getting a smaller query